harmonia/src/config.rs

```diff
@@ -33,8 +34,9 @@ pub(crate) struct Config {
     pub(crate) priority: usize,
     #[serde(default)]
     pub(crate) sign_key_path: Option<String>,
-    #[serde(default)]
+
     pub(crate) secret_key: Option<String>,
+    pub(crate) store: Store,
```

Comment:
```
vm-test-run-t00-simple> harmonia # [    5.509658] harmonia[793]: [2024-04-18T10:26:25Z ERROR harmonia] Couldn't parse config file '/nix/store/0y2sph6l7nh5qvxjglkb1c8hqhbkxwsz-harmonia.toml'
vm-test-run-t00-simple> harmonia # [    5.510624] harmonia[793]: [2024-04-18T10:26:25Z ERROR harmonia] because: TOML parse error at line 1, column 1
vm-test-run-t00-simple> harmonia # [    5.511463] harmonia[793]:       |
vm-test-run-t00-simple> harmonia # [    5.511927] harmonia[793]:     1 | bind = "[::]:5000"
vm-test-run-t00-simple> harmonia # [    5.512993] harmonia[793]:       | ^^^^^^^^^^^^^^^^^^
vm-test-run-t00-simple> harmonia # [    5.513330] harmonia[793]:     missing field `store`
vm-test-run-t00-simple> harmonia # [    5.513694] harmonia[793]:
vm-test-run-t00-simple> harmonia # [    5.519729] systemd[1]: harmonia-dev.service: Main process exited, code=exited, status=1/FAILURE
vm-test-run-t00-simple> harmonia # [    5.520837] systemd[1]: harmonia-dev.service: Failed with result 'exit-code'.
```

Seems this breaks the config parser, probably because it's a "smart class" which generates a TOML parser for every field